src/model/CoilCoolingDXSingleSpeed.cpp

```diff
@@ -897,18 +923,21 @@ namespace model {
     setEnergyInputRatioFunctionOfFlowFractionCurve(energyInputRatioFofFlow);
     setPartLoadFractionCorrelationCurve(partLoadFraction);
     setEvaporativeCondenserEffectiveness(0.0);
-    setEvaporativeCondenserAirFlowRate(boost::none);
-    setEvaporativeCondenserPumpRatedPowerConsumption(boost::none);
+    autosizeEvaporativeCondenserAirFlowRate();                //autosize
+    autosizeEvaporativeCondenserPumpRatedPowerConsumption();  //autosize
     setCrankcaseHeaterCapacity(0.0);
     setMaximumOutdoorDryBulbTemperatureForCrankcaseHeaterOperation(0.0);
     setMinimumOutdoorDryBulbTemperatureforCompressorOperation(-25.0);  // Per E+ IDD default
-    //setSupplyWaterStorageTankName("");
-    //setCondensateCollectionWaterStorageTankName("");
+    setNominalTimeForCondensateRemovalToBegin(0.0);
+    setRatioOfInitialMoistureEvaporationRateAndSteadyStateLatentCapacity(0.0);
+    setMaximumCyclingRate(0.0);
+    setLatentCapacityTimeConstant(0.0);
+    setCondenserType("AirCooled");
+    setEvaporativeCondenserEffectiveness(0.9);
+    setCrankcaseHeaterCapacity(0.0);
+    setMaximumOutdoorDryBulbTemperatureForCrankcaseHeaterOperation(10.0);
```

Comment:
setCrankcaseHeaterCapacity is called twice.

setMaximumOutdoorDryBulbTemperatureForCrankcaseHeaterOperation is called twice, one with 0.0 (historical) and later with 10.0. You're also changing a default, and not doing the same on CoilCoolingDXTwoSpeed